src/core/sort.js

```diff
@@ -140,6 +146,35 @@ class Sort {
         return this;
     }
 
+    /**
+     * Note: This method is uncompatible with elasticsearch 6.0 and older.
```

Comment:
Minor type - `{uncompatible => incompatible}`. 

Please move the note below just above the example.

Reply:
done

Reply:
Could you please correct the typo? "uncompatible" is not a valid word as far as I know.

Reply:
Sorry, missed commit :)

Reply:
No problem.